include/deal.II/distributed/tria_base.h

```diff
@@ -506,421 +502,6 @@ namespace parallel
     DEAL_II_DEPRECATED
     virtual void
     load(const std::string &filename, const bool autopartition) = 0;
-
-    /**
-     * Register a function that can be used to attach data of fixed size
-     * to cells. This is useful for two purposes: (i) Upon refinement and
-     * coarsening of a triangulation (@a e.g. in
-     * parallel::distributed::Triangulation::execute_coarsening_and_refinement()),
```

Comment:
Where did all of these functions go? Unless these functions went to the base class, this is all incompatible, right?

Reply:
I've_ moved everything over to the parent class `Triangulation<dim, spacedim>`.

The only exceptions are the classes `DataTransfer` and `CellDataAttach`, which by the way weren't meant to be used externally anyways. So, this changes shouldn't break compatibility.